### PR TITLE
PLG-133: remove old dashboard related code

### PR DESCRIPTION
**Pull request by rouliane**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: tests/back/Platform/Specification/VersionProviderSpec.php [hunk 1]
```diff
@@ -16,7 +16,7 @@ function it_provides_ce_edition()
     function it_provides_ce_version()
     {
         $this->beConstructedWith(StaticCommunityVersion::class);
-        $this->getEdition()->shouldReturn('12.42.20-BETA2');
+        $this->getEdition()->shouldReturn('CE');
```
Comment (AymericPlanche) [inline tests/back/Platform/Specification/VersionProviderSpec.php:19]: nope

```suggestion
        $this->getVersion()->shouldReturn('12.42.20-BETA2');
```

File: tests/back/Pim/Enrichment/Specification/Bundle/Widget/CompletenessWidgetSpec.php [hunk 2]
```diff
@@ -27,11 +27,6 @@ function it_has_an_alias()
         $this->getAlias()->shouldReturn('completeness');
     }
 
-    function it_exposes_the_completeness_widget_template()
```
Comment (AymericPlanche) [inline tests/back/Pim/Enrichment/Specification/Bundle/Widget/CompletenessWidgetSpec.php:30]: the entire spec could be deleted no ? (same for other files)

File: tests/back/Platform/Specification/VersionProviderSpec.php (hunk 1 above)
Comment (rouliane) [reply tests/back/Platform/Specification/VersionProviderSpec.php]: Oups... Fixed

**Review by LaurentPetard — APPROVED** [review]

**Review by olivierpontier — APPROVED** [review]

**Comment by olivierpontier** [comment]
You have some issues during the front static validation step on the CI workflow

**Pull request merged**